packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Basic.Example.tsx

```diff
@@ -27,6 +27,45 @@ export class ComboBoxBasicExample extends React.Component<any, any> {
   { key: 'J', text: 'Option j' },
   ];
 
+  private _testOptionsWithCustomStyling =

```

Comment:
I think each of the examples should only return one ComboBox. Probably best to just move this to a ComboBox.CustomStyledOptions.Example.tsx